src/api/json/catalog.json

```diff
@@ -88,6 +88,12 @@
       "fileMatch": [ ".babelrc" ],
       "url": "http://json.schemastore.org/babelrc"
     },
+    {
+      "name": ".backportrc.json",
+      "description": "Backport configuration file",
+      "fileMatch": [ ".backportrc.json" ],
+      "url": "http://json.schemastore.org/backport"
```

Comment:
Just assumed this would be the url

Reply:
... but looking closer, perhaps it should be `http://json.schemastore.org/backportrc` ?